src/vcpkg/base/unicode.cpp

```diff
@@ -214,11 +184,10 @@ namespace vcpkg::Unicode
 
     struct Utf8Category : std::error_category
     {
-        const char* name() const noexcept override;
+        inline const char* name() const noexcept override { return "utf8"; }
```

Comment:
```suggestion
        const char* name() const noexcept override { return "utf8"; }
```

Inline is implied because you made it part of the class' body.